web/package.json

```diff
@@ -28,7 +28,8 @@
     "flow:install-types": "flow-typed install --skip"
   },
   "dependencies": {
-    "@babel/polyfill": "^7.8.3",
+    "@babel/polyfill": "^7.12.1",
+    "@babel/runtime": "^7.12.5",
```

Comment:
Why do we need babel runtime here?

Reply:
Yes the webapp should have this runtime as dependency

Reply:
Yeah okay, but was is the reason why we need this? :D

Reply:
before this PR it probably was used inexplicitly. I added it for consistency.